### PR TITLE
CI-pipeline: check syntax of ttl files and check examples against schema

### PR DESCRIPTION
**Pull request by robertschubert**
- add syntax check
- add schema check of instance files
- refactor README.md
- add known issues of SD-Creation-Wizard.
- hdmap_instance.json is conform now.

After this one is merged, we will create an issue for all example files which are not conform to the shacl.

### CODE REVIEW

File: .github/workflows/branch-push.yml [hunk 1]
```diff
@@ -44,4 +45,81 @@ jobs:
...
+
+      - name: Set up Python
+        if: env.TTL_HAS_CHANGED == 'true'
+        uses: actions/setup-python@v5
+        with:
+          python-version: 3.12.0
+
+      - name: Install rdflib
+        if: env.TTL_HAS_CHANGED == 'true'
+        run: pip install rdflib
```
Comment (jdsika) [inline .github/workflows/branch-push.yml:79]: Can we add the requirements to a file in order to install conviniently locally?
```
    - name: Install dependencies
      run: pip install -r requirements_develop.txt
```

File: README.md [hunk 2]
```diff
@@ -123,6 +117,18 @@ This section describes guidelines that _must_ be followed when applying changes
...
+The CI/CD pipeline is defined in the `.github/workflows` directory. The pipeline is triggered on every push to the repository as defined in the workflow. 
+ 
+### Generate VARIABLES.md (overview of used properties in the SHACL files)
+The file VARIABLES.md will be generated automatically when a push to a non-main branch is executed. This file is existent in every subdirectory once there is a SHACL file containing properties. This should help to get a fast overview of the properties used in the SHACL files.
+
+>NOTE: the VARIABLES.md file should not be changed since it will be overwritten automatically. 
+
+### Check syntax of Turtle files
+The pipeline checks the syntax of the Turtle files (`*.ttl`) by loading a RDF graph. If the Turtle file is not correct the pipeline fails with a detailed error message.
+
```
Comment (jdsika) [inline README.md:131]: Should we add notes to execute the check locally using python venv?
```
$ python3 -m venv .venv
$ source .venv/bin/activate
(.venv) $ python3 -m pip install --upgrade pip
(.venv) $ python3 -m pip install -r requirements_develop.txt
# execute check from CI
```

**Review by jdsika — COMMENTED** [review]
See notes in the code<<

File: .github/workflows/branch-push.yml (hunk 1 above)
Comment (robertschubert) [reply .github/workflows/branch-push.yml]: added requirements_ci.txt and linked it in the ci

File: README.md (hunk 2 above)
Comment (robertschubert) [reply README.md:133]: added this into the readme

**Review by jdsika — APPROVED** [review]

**Comment by robertschubert** [comment]
merge with bypass (without Lena's review)

**Pull request merged**